test/karma/package.json

```diff
@@ -1,26 +1,28 @@
 {
   "private": true,
-  "main": "test-dist/index.cjs.js",
-  "module": "test-dist/index.js",
-  "types": "test-dist/types/components.d.ts",
-  "collection": "test-dist/collection/collection-manifest.json",
+  "main": "test-output/test-dist/index.cjs.js",
+  "module": "test-output/test-dist/index.js",
+  "types": "test-output/test-dist/types/components.d.ts",
+  "collection": "test-output/test-dist/collection/collection-manifest.json",
   "scripts": {
+    "clean": "rm -rf test-output tmp-compiled-tests",
+    "start": "node ../../bin/stencil build --dev --watch --serve --es5",
     "build.app": "npm run compile.test-app && node ../../bin/stencil build --debug --es5",
     "compile.test-app": "node ../../node_modules/typescript/lib/tsc -p tsconfig.json",
-    "build.prerender": "node ../../bin/stencil build --config test-prerender/stencil.config-prerender.ts --prerender --debug && node test-prerender/no-script-build.js",
-    "build.sibling": "node ../../bin/stencil build --config test-sibling/stencil.config.ts --debug",
+    "build.custom-elements": "webpack-cli --config test-app/custom-elements-output-webpack/webpack.config.js && webpack-cli --config test-app/custom-elements-output-tag-class-different/webpack.config.js && webpack-cli --config test-app/custom-elements-delegates-focus/webpack.config.js",
```

Comment:
I still need to come up with a more elegant solution here, but splitting this command out seemed like a good first step